network-api/networkapi/wagtailpages/pagemodels/customblocks/about_block.py

```diff
@@ -0,0 +1,14 @@
+from wagtail import blocks
+from wagtail.images.blocks import ImageChooserBlock
+
+from .full_content_rich_text_options import full_content_rich_text_options
+
+
+class AboutBlock(blocks.StructBlock):
+    image = ImageChooserBlock()
+    image_alt_text = blocks.CharBlock(required=True, help_text="Image description (for screen readers).")
+    heading = blocks.CharBlock(help_text="Heading for the About Block")
+    content = blocks.RichTextBlock(features=full_content_rich_text_options)
```

Comment:
Do we need all the features here? Since this is for a paragraph, I would be more inclined to actually restrict the features to just italics, bolds and maybe links? I think this would be a case of doing ["faster horses"](https://torchbox.com/blog/rich-text-fields-and-faster-horses/) but I would be the happy to discuss this.

Reply:
Hi @jhonatan-lopes this makes sense to me! 

I have updated this field to instead use a list we call `base_rich_text_options` which includes only Bold, Italics, and Links 👍